app/views/partials/personal_key/_entry_fields.html.slim

```diff
@@ -5,4 +5,5 @@
     - Figaro.env.recovery_code_length.to_i.times do
       .col.col-3.px1.sm-px2
         = f.input attribute_name, as: :array, label: false,
-          input_html: { required: true, 'aria-labelledby': 'personal-key-label' }
+          input_html: { required: true, autocapitalize: 'none', autocomplete: 'off', \
+            spellcheck: 'false', 'aria-labelledby' => 'personal-key-label' }
```

Comment:
I could not figure out how to make our slim lint happy, it told me to indent by 2 and I did and it was still mad. I will alpha these in the meantime